mycore-base/src/main/resources/config/mycore.properties

```diff
@@ -28,6 +28,12 @@
   MCR.baseurl=http://localhost:8291
 
 ##############################################################################
+# Google Sitemap                                                             #
+##############################################################################
+# Sets the filter for the Solr query
+  MCR.GoogleSitemap.SolrQuery=(objectType:mods AND -state:*) OR (objectType:mods AND state:published)
```

Comment:
Why is that in mycore-base? mycore-base does not know anything about SOLR or site maps or even MODS.